src/styles/Pomodoro.css

```diff
@@ -0,0 +1,12 @@
+.pomodoro {
+    position: absolute;
+    width: 100vw;
+    height: 105vh;
```

Comment:
Generally speaking you never need to go above 100vh/vw for styling. This is a work in progress at the minute though so Im aware theres still things to be worked out